lib/pg_rls/admin.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module PgRls
+  # The admin module provides the admin_execute method to execute
+  # queries on the admin shard.
+  module Admin
+    def self.admin_execute(sql = nil)
+      PgRls::Record.connected_to(shard: :admin) do
+        return yield.presence if block_given?
+
+        PgRls::Record.connection.execute(sql).presence
+      end
+    end
+  end
+end
```

Comment:
there is a class name PgRls::Tenant which is an ActiveRecord, it would be nice if we could have like PgRls::Admin, so it would be in the same folder model/tenant/admin or something like that instead of a module that it;s added again in the main lib?